gpsd_format/drivers.py

```diff
@@ -214,24 +201,39 @@ def read(self, size):
         return self.obj.read(size)
 
 
-class NewlineJSON(BaseDriver):
+class FileDriver(BaseDriver):
```

Comment:
But you're not using it with either one of those drivers.  It's only being used with GZIP, which was working without issue when it subclassed BaseDriver.